docs/diagrams/StorageClassDiagram.puml

```diff
@@ -37,7 +40,10 @@ Storage -right-|> AddressBookStorage
 JsonUserPrefsStorage .up.|> UserPrefsStorage
 JsonAddressBookStorage .up.|> AddressBookStorage
 JsonAddressBookStorage ..> JsonSerializableAddressBook
-JsonSerializableAddressBook --> "*" JsonAdaptedPerson
-JsonAdaptedPerson --> "*" JsonAdaptedTag
+JsonSerializableAddressBook --> "0..*" JsonAdaptedPerson
+JsonAdaptedPerson --> "0..*" JsonAdaptedTag
+JsonAdaptedPerson --> "0..*" JsonAdaptedWedding
+JsonAdaptedVendor --|> JsonAdaptedPerson
```

Comment:
I don't think this is right, @tingxuanp 's PR that adds the vendor class does not have `JsonAdaptedVendor` extend `JsonAdaptedPerson` ?